lib/mongo/auth/aws/request.rb

```diff
@@ -215,7 +215,6 @@ def canonical_request
         # @return [ String ] The signature.
         def signature
           hashed_canonical_request = Digest::SHA256.hexdigest(canonical_request)
-          hashed_body = Digest::SHA256.new.update(STS_REQUEST_BODY).hexdigest
```

Comment:
I think this ended up being in the hashed canonical request.

Reply:
I am wondering how is it possible. We create a new digest here, and it is not used anywhere else.

Reply:
The body is part of canonical request. This line is likely a leftover from the in-progress implementation of aws auth.